src/lib.rs

```diff
@@ -152,7 +173,38 @@ impl ApplicationBuilder {
         self.log_settings.everything = b; self
     }
 
-    pub fn start (&mut self) {
+    pub fn start (&mut self, args: Option<Vec<String>>) {
+
+        let mut init_path: Option<PathBuf> = None;
+        let mut init_args: Option<Vec<String>> = None;
+        let mut package_path: Option<String> = None;
+
+        match args {
+            // Interpreter
+            Some(args) => {
+
+                init_path = Path::new(args.first().expect("Missing first argument."))
```

Comment:
Same as panic, you do want to avoid using expect, but for now you can leave it in for the time being unless you feel like switching it out. In my opinion, you should move `args.first()` outside of the `Path::new` function and use a match statement.

Reply:
First argument is guaranteed by `clap` I think.

Reply:
I havent tested the code to confirm it but its I will let it through